gradle/scripts/defaultBuildProperties.gradle

```diff
@@ -28,7 +28,7 @@ def BuildProperties BUILD_PROPERTIES = new BuildProperties(project)
     .register(new BuildProperty("doNotSignArtifacts", false, "Do not sight Maven artifacts"))
     .register(new BuildProperty("gobblinFlavor", "standard", "Build flavor (see http://gobblin.readthedocs.io/en/latest/developer-guide/GobblinModules/)"))
     .register(new BuildProperty("hadoopVersion", "2.6.0", "Hadoop dependencies version"))
```

Comment:
Updated to 2.3.0 in my revision. FYI, Hive 1.0.1 depends on Hadoop 2.5.0: https://repo1.maven.org/maven2/org/apache/hive/hive/1.0.1/hive-1.0.1.pom